docs/ASH_AND_ELIXIR_IMPROVEMENT_PLAN.md

```
@@ -0,0 +1,1351 @@
+end
+
+def handle_info({:create, %Profile{} = profile}, socket) do
+  {:noreply, add_profile_to_assigns(socket, profile)}
+end
+
+def handle_info({:destroy, %Profile{id: id}}, socket) do
+  {:noreply, remove_profile_from_assigns(socket, id)}
+end
+```
+
+### Acceptance Criteria
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)